### PR TITLE
Pspec check

### PR DESCRIPTION
**Pull request by aelanman**
Introduces a new unit test that checks that a flat-sky visibility simulation has the correct power spectrum amplitude from a simple delay spectrum estimator. This also adds a handful of cosmology related functions originally pulled from capo.

### CODE REVIEW

File: healvis/cosmology.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    if isinstance(z, np.ndarray):
+        if isinstance(omega, np.ndarray):
+            z, omega = np.meshgrid(z, omega)
+        elif isinstance(dnu, np.ndarray):
+            z, dnu = np.meshgrid(z, dnu)
+    elif isinstance(dnu, np.ndarray) and isinstance(omega, np.ndarray):
+        dnu, omega = np.meshgrid(dnu, omega)
+    nu0 = 1420. / (z + 1) - dnu / 2.
+    nu1 = nu0 + dnu
+    dz = 1420. * (1 / nu0 - 1 / nu1)
```
Comment (nkern) [inline healvis/cosmology.py]: let's use `f21` here

File: healvis/cosmology.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+# -----------------------
+#  Functions and constants related
+#  to conversions between instrument and
+#  cosmological units.
+# -----------------------
+
+f21 = 1.420405751e9
+c_ms = c.to('m/s').value
+pi = np.pi
```
Comment (nkern) [inline healvis/cosmology.py]: let's just get rid of this and use `np.pi` everywhere for simplicity

File: healvis/cosmology.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+def jy2Tsr(f, bm=1.0, mK=False):
```
Comment (nkern) [inline healvis/cosmology.py]: not sure why this was moved to `cosmology` b/c it doesn't require any notion of a cosmology to work properly

File: healvis/cosmology.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    k_boltz = 1.380658e-16   # erg/K
+    lam = c_cmps / f  # cm
+    fac = 1.0
+    if mK:
+        fac = 1e3
+    return 1e-23 * lam**2 / (2 * k_boltz * bm) * fac
+
+
+def dL_df(z):
+    '''[Mpc]/Hz, from Furlanetto et al. (2006)'''
```
Comment (nkern) [inline healvis/cosmology.py]: Is this comoving or physical Mpc? I think its comoving. If it is, let's be specific via `[cMpc / Hz]`

File: healvis/cosmology.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    c_kms = 299792.458  # km/s
+    return c_kms / (cosmo.H0.value * cosmo.efunc(z)) * (z + 1)**2 / f21
+
+
+def dL_dth(z):
```
Comment (nkern) [inline healvis/cosmology.py]: same here

File: healvis/cosmology.py (hunk 3 above)
Comment (aelanman) [reply healvis/cosmology.py]: True, but this way I don't have to import from astropy.constants in utils. I wasn't sure what to call this file exactly, but I'm thinking of it as "any functions needed to go between sky and instrument units".

Comment (nkern) [reply healvis/cosmology.py]: okay, I'm fine with multiple imports of astropy.constants in separate modules. I think having it in utils makes sense b/c it is an in between kind of function that can be used by different modules. I guess my point is, if we keep `cosmology.py` as the module name then lets move the `jy2Tsr` out of it, as someone looking for this functionality might not look in `cosmology.py` to find it

Comment (aelanman) [reply healvis/cosmology.py]: Fair enough. I'll move it back to utils.

**Pull request merged**